tests/components/dte_energy_bridge/test_sensor.py

```diff
@@ -1,65 +1,55 @@
 """The tests for the DTE Energy Bridge."""
-
-import unittest
-
 import requests_mock
 
-from homeassistant.setup import setup_component
-
-from tests.common import get_test_home_assistant
+from homeassistant.setup import async_setup_component
 
 DTE_ENERGY_BRIDGE_CONFIG = {"platform": "dte_energy_bridge", "ip": "192.168.1.1"}
 
 
-class TestDteEnergyBridgeSetup(unittest.TestCase):
-    """Test the DTE Energy Bridge platform."""
+async def test_setup_with_config(hass):
+    """Test the platform setup with configuration."""
+    assert await async_setup_component(
+        hass, "sensor", {"dte_energy_bridge": DTE_ENERGY_BRIDGE_CONFIG}
+    )
```

Comment:
we have to block till everything is done after this.

Reply:
Done.